gcc/testsuite/rust/compile/torture/extern_mod4.rs

```diff
@@ -0,0 +1,17 @@
+// { dg-additional-options "-w" }
+// { dg-output "12" }
```

Comment:
As this is a `compile` test, it won't be executed, and you'll not see the `12` output.  Not sure if this is worth turning into an `execute` test?  (If yes, then also need to add the `return 0` hack at end of `fn main`.)

Reply:
Ah, right, thanks! I'll open a PR to fix this